web/src/util/validation.js

```diff
@@ -0,0 +1,12 @@
+export const MESSAGES = {
+  required: 'Please fill out this field',
+  email: 'Please enter a valid email address'
+};
+
+export const required = val =>
+  val === '' || val === null ? MESSAGES.required : null;
+
+export const email = val =>
+  val && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(val)
```

Comment:
Email validation!  😬  What about non-ascii or emoji domains?!  (About 80% kidding, but email validation always makes me squirm because it's so freakin' complicated if you want to capture every possible valid case.)

Reply:
i agree, we should use a library